manifests/pipecd/templates/configmap.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
     {{- include "pipecd.labels" . | nindent 4 }}
     grafana_dashboard: "1"
   annotations:
-    k8s-sidecar-target-directory: "/tmp/dashboards/cluster"
+    k8s-sidecar-target-directory: "/tmp/dashboards/Cluster"
```

Comment:
Any specific reason for this rename? 🤔

Reply:
Without renaming this, the dashboard directories become like this comment 😢 
https://github.com/pipe-cd/pipecd/pull/5103#issuecomment-2264301684

Reply:
Got it, thank you 👍